workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/IReader.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Improbable.Worker;
+using Unity.Entities;
+
+namespace Improbable.Gdk.Core.MonoBehaviours
+{
+    /// <summary>
+    ///     The Reader interface is used by MonoBehaviours to query information about a SpatialOS component and register
+    ///     callbacks for updates, events and commands for component instances.
+    /// </summary>
+    /// <typeparam name="TComponent">The data type for the SpatialOS component.</typeparam>
+    /// <typeparam name="TComponentUpdate">The data type for the SpatialOS component's update.</typeparam>
+    public interface IReader<TComponent, TComponentUpdate>
```

Comment:
Do we need the second generic here?

Reply:
Probably could be removed, and where it's used instead have ISpatialComponentUpdate<TComponent> unless I'm missing something?

Reply:
Aye, not necessary, doing the recommended change